openpype/hosts/max/plugins/publish/validate_viewport_camera.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.max.api.lib import get_current_renderer
+
+from pymxs import runtime as rt
+
+
+class ValidateViewportCamera(pyblish.api.InstancePlugin,
+                             OptionalPyblishPluginMixin):
+    """Validates Viewport Camera
```

Comment:
Why do we need to validate the viewport camera? for what?

Reply:
It seems very odd that the viewport must be set to a viewport to be able to do a 'batch' render.
I see in other areas of the code that you're able to set e.g. an explicit "Camera" option for Mtoa - which as such seems like it wouldn't need the viewport to specify a camera, yes?

Similarly the 3dsmax Deadline Submitter has an explicit 'Camera' field:
![image](https://github.com/ynput/OpenPype/assets/2439881/12511f35-f3c2-4a0b-9526-c9a1a8d40a75)

Why can't we use that plug-in info in the submission to specify the camera for the batch render? Would mean there's no need to tweak the scene at all. Win-win?